keyword_spotting_data_generator/evaluation/extractor/edit_distance_extractor.py

```diff
@@ -63,8 +63,13 @@ def extract_keywords(self, data, window_ms=1000, hop_ms=250):
 
         mfcc_audio = self.compute_mfccs(data)
         whitened = whiten(mfcc_audio)
-        K = 100
-        code_book = kmeans(whitened, K)[0]
+
+        k = whitened.shape[0]
+        code_book, distortion = kmeans(whitened, k)
+
+        while distortion < 1:
```

Comment:
Can we have a parameter for this threshold? something like `distortion_threshold`?
https://github.com/castorini/honk/blob/279af81b6e9a874704373aac01e7d1caca40f708/keyword_spotting_data_generator/evaluation/extractor/edit_distance_extractor.py#L8